backend/app/services/subscription/helpers.py

```
@@ -0,0 +1,471 @@
+    workspace_name = f"{git_repo.replace('/', '-')}-{branch_name}".lower()[:100]
+    namespace = "default"
+
+    # Check if workspace already exists
+    existing = (
+        db.query(TaskResource)
+        .filter(
+            TaskResource.user_id == user_id,
+            TaskResource.kind == KIND_WORKSPACE,
+            TaskResource.name == workspace_name,
+            TaskResource.namespace == namespace,
+            TaskResource.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `TaskResource.is_active:` for truth checks

Replace with `TaskResource.is_active`

(E712)